input/blog/posts/net-foundation-election-2021-nominations-open.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: .NET Foundation Election 2021 Nominations are Open!
+Author: Iris Classon
+Published: 2021-07-14
+---
+
+## .NET Foundation Election 2021 Nominations are Open!
+
+It’s that time of the year again dear friends- the .NET nominations! On behalf of the board and nomination committee we would like to welcome all of you to participate- be that by [applying as a candidate yourself](https://forms.office.com/Pages/ResponsePage.aspx?id=8ru98HbNZ0mMTFCoAnRpEmp3qxxG5iNBl2dHcSfTzztUOEdKT05ENERURFFIMTFOSDU3TE81NFJDQi4u), suggesting somebody else or simply preparing to vote and help spread the word.
+
+As a board member you get to influence and shape the future of one of the biggest platforms there is, .NET and everything it entails. You won’t be doing this alone, you’ll be working side by side with other passionate souls on the board, committees, TSG, Microsoft, community members and more. The [expectations for board members are outlined here](https://dotnetfoundation.sharepoint.com/:b:/g/EVpjN2ykcKJGkfPjbwbTNIUB_WOVtDAEElC-xXCmEh5QzQ)
```

Comment:
Suggested:

"As a board member, you get to influence and shape the future of one of the biggest platforms there is—.NET—and everything it entails. You won't be doing this alone. You'll be working side by side with other passionate souls on the board, committees, TSG, Microsoft, community members, and more. You can review the expectations for board members."